index.html

```diff
@@ -0,0 +1,152 @@
+          people from over 80 countries is taking place in October, in Korea.
+        </h3>
+        <h2></h2>
+      </div>
+    </section>
+    <section class="program">
+        <h2 class="section-heading">Main Program</h2>
+            <div class="red-bar"></div>
+            <div class=" main-prog-items">
+                <div class="prog-items">
+                    <img src="./Images/program1.png" alt="icon">
+                    <h2>Lecture&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program2.png" alt="icon">
+                    <h2>CC Exhibition</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program3.png" alt="icon">
+                    <h2>Forum&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program4.png" alt="icon">
+                    <h2>Workshop</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program5.png" alt="icon">
+                    <h2>CC Lingration</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+            </div>
+            <button type="submit" class="register-btn">Register Now</button>
+            <a href="#" id="whole-program">See The Whole Program</a>
+    </section>
+    <section class="speaker_section" id="speaker_section">
+      <h2 class="section-heading_1">Featured Speakers</h2>
+      <div class="red-bar_1"></div>
+      <div id="featured-speakers"></div>
+    </section>
+    <section class="partners-section">
+      <h2 class="section-heading">Partners</h2>
+      <div class="red-bar"></div>
+      <div class="partners-flex">
+        <div class="partners-list">
+          <img class="sponsor" src="./Images/partner1.png" alt="logo">
+          <img class="sponsor" src="./Images/partner2.png" alt="logo">
+          <img class="sponsor" src="./Images/partner3.png" alt="logo">
+          <img class="sponsor" src="./Images/partner4.png" alt="logo">
+          <img class="sponsor" src="./Images/partner5.png" alt="logo">
+        </div>
+      </div>
+    </section>
+    <footer class="footer-rights">
+        <div>
+          <a href="#">
+            <img class="logo-2" src="Images/logo-black.png" alt="logo" />
+          </a>
+        </div>
+        <div>
```

Comment:
- [ ] kindly reduce the space between your footer logo and the text like in the original version. for both home and about page
yours
![Screenshot (982)_LI](https://user-images.githubusercontent.com/65298048/233436734-5bb64aac-f199-4a84-8d8f-e45a99798e15.jpg)
original
![Screenshot (981)](https://user-images.githubusercontent.com/65298048/233436825-c15e1abd-3bbf-49ce-8102-6dfb7dbd75dd.png)